src/utl/include/utl/Logger.h

```diff
@@ -233,9 +243,15 @@ class Logger
     inline void log_metric(const std::string_view metric,
                            const Value& value)
     {
+      std::string key;
+      if (metrics_stages_.empty()) 
+        key = std::string(metric);
```

Comment:
Why is std::string required?  std::string::operator= should work with a std::string_view